add git cheat sheet to docs

sophiad96 opened the pull request:
## Description & motivation
This PR adds a git cheat sheet page (actually called The IDE Git Button) that provides users with guidance on what the different options that appear on the git button in the IDE will do.


## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [X] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [X] The page has been added to `website/sidebars.js`
- [X] The new page has a unique filename

sophiad96 committed:
add git cheat sheet to docs

sophiad96 commented:
The images in this show as broken. Couldn't figure out why.

clrcrl commented on website/docs/docs/dbt-cloud/cloud-ide/the-ide-git-button.md:
```diff
@@ -0,0 +1,43 @@
+**Initialize your project**: This is done when first setting up your project. Initializing a project will create all required directories and files within an empty repository. Note: This option will not display if your repo isn't completely empty (i.e. includes a README file).
+
+<p align="center">
+<Lightbox src="/img/initialize.png" />
```
re: the images are broken

You'll have to add the images to the repo as well — I think I go over that in the Loom vid in the README :)

For example, if I have an image reference like this:
```
<Lightbox src="/img/create-bigquery-project.png" title="Create a new GCP project" />
```
It picks up the image here: https://github.com/fishtown-analytics/docs.getdbt.com/tree/current/website/static/img

sophiad96 replied:
ah tysm!

sophiad96 committed:
fix broken images

drewbanin commented:
removing myself from review, adding @saouderkirk

annafil committed:
Delete package-lock.json

triggers a merge conflict. We probably want the version of this that is already checked in

annafil committed:
yup that last commit was not what I wanted. such is life

annafil committed:
Merge branch 'current' into add-git-cheat-sheet

annafil reviewed (approved):
LGTM! 🎉 

Non-blocking suggestion: I would love to see these docs point to a really good Git reference -- e.g. "Not sure what these terms mean? Check out X for more details" 

I personally like [this one](https://rogerdudler.github.io/git-guide/) but you probably have your own favorite :)
